package.json

```diff
@@ -31,5 +33,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "cross-env": "^7.0.3"
```

Comment:
i would put this as a dependency just in case for some reason, during deployment, i end up using a windows environment for the server (very rare, but possible)